code/model/modelread.cpp

```diff
@@ -5431,14 +5487,16 @@ void swap_bsp_data( polymodel * pm, void * model_ptr )
 
 void swap_sldc_data(ubyte * buffer)
 {
+//ShivanSpS - Changed type char for a type int for SLC2
 #if BYTE_ORDER == BIG_ENDIAN
```

Comment:
Wait, we still support Big endian? Do we actually need that for ARM? I thought that could also handle little endian. If that is the case, maybe just remove this.

Reply:
Well, the function of swaping SLDC data is already in the code, so im forced to adjust it to the new node format, otherwise the game will crash on a big endian system.

Reply:
Yes but the point is if the big endian case is even still necessary since I think all of our supported systems are little endian. That is not a question to answer here though so there is nothing to change here. It was just something I noticed.